### PR TITLE
Adding contributing docs

### LINKED ISSUE #255: [docs] contribution guildelines
The `django-freeradius` docs has it's own contribution page [here](https://django-freeradius.readthedocs.io/en/latest/general/contributing.html).
Which makes no mention of the contribution page of the organization [here](http://openwisp.io/docs/developer/contributing.html)

We want to mention the organization guidelines in the `django-freeradius` documentation and remove any duplicate documentation from the django-freeradius contribution page.

(Remember to read https://django-freeradius.readthedocs.io/en/latest/general/contributing.html#update-the-documentation)

### PR DESCRIPTION
**Pull request by aviral2004**
Added a section asking to follow the OpenWisp contributing guidelines.
Will resolve #255

### CODE REVIEW

File: docs/source/general/contributing.rst [hunk 1]
```diff
@@ -10,17 +10,24 @@ Follow these guidelines to speed up the process.
   :backlinks: none
   :depth: 3
 
+Note for contributors
```
Comment (atb00ker) [inline docs/source/general/contributing.rst]: I may have made a mistake in explaining,
I meant: 
```
.. note::
    Before continuing with your contributing endeavours, please read the 
    OpenWISP contributing guidelines to ensure smooth working of the
    freeradius repository.
    The guidelines can be found at the following `link here <http://openwisp.io/docs/developer/contributing.html>`_

```

File: docs/source/general/contributing.rst [hunk 2]
```diff
@@ -10,17 +10,24 @@ Follow these guidelines to speed up the process.
...
+---------------------
+
+Before continuing with your contributing endeavours, please read the
+OpenWISP contributing guidelines to ensure smooth working of the
+freeradius repository.
+
+The guidelines can be found at the following link:
+<http://openwisp.io/docs/developer/contributing.html>
+
 Reach out before you start
```
Comment (atb00ker) [inline docs/source/general/contributing.rst]: This is repeated, can you please remove this section.

File: docs/source/general/contributing.rst [hunk 3]
```diff
@@ -69,15 +76,6 @@ When you introduce changes, ensure test coverage is not decreased with:
 
 Follow style conventions (PEP8, isort, JSLint)
 ----------------------------------------------
-
-First of all, install the test requirements:
```
Comment (atb00ker) [inline docs/source/general/contributing.rst]: Why was this section removed, it's not in the organization docs, is it?

File: docs/source/general/setup.rst [hunk 4]
```diff
@@ -128,8 +128,64 @@ In case you are using `openwisp-radius <https://github.com/openwisp/openwisp-rad
     ./manage.py migrate --fake openwisp_radius 0001_initial_freeradius
     ./manage.py migrate
 
-Installing for development
```
Comment (atb00ker) [inline docs/source/general/setup.rst]: Actually, I think the sections that remain in the `Contribution.rst` can be moved to `Setup.rst` because they are specific to the setup of freeradius, wouldn't you agree? :smile:

File: docs/source/general/setup.rst [hunk 5]
```diff
@@ -128,8 +128,64 @@ In case you are using `openwisp-radius <https://github.com/openwisp/openwisp-rad
...
+
+.. code-block:: shell
+
+    cd tests/
+    pwd
+
+More information can be found at the
+`management commands page <./management_commands.html>`_.
+
+Note: Development setup
```
Comment (atb00ker) [inline docs/source/general/setup.rst]: Can you remove `Note:` here?

File: docs/source/general/setup.rst (hunk 4 above)
Comment (aviral2004) [reply docs/source/general/setup.rst]: I couldn't understand what you meant by this. Can you please explain what you mean by the sections that remain.

File: docs/source/general/contributing.rst (hunk 2 above)
Comment (atb00ker) [reply docs/source/general/contributing.rst]: @Aviral14 this needs attention too, "Reach out before you start" step is here as well: http://openwisp.io/docs/developer/contributing.html as "Introduce yourself"

File: docs/source/general/setup.rst (hunk 4 above)
Comment (atb00ker) [reply docs/source/general/setup.rst]: I mean that the content from `general/contributing.rst` can be moved to `setup.rst` because only repository setup related steps are remaining, right? :smile:

File: docs/source/general/contributing.rst [hunk 6]
```diff
@@ -74,9 +39,9 @@ First of all, install the test requirements:
 
 .. code-block:: shell
 
-    workon radius  # activate virtualenv
-    pip install --no-cache-dir -U -r requirements-test.txt
-    npm install -g jslint
+     workon radius  # activate virtualenv
+     pip install --no-cache-dir -U -r requirements-test.txt
+     npm install -g jslint
```
Comment (nemesisdesign) [inline docs/source/general/contributing.rst]: do not add these spaces please

File: docs/source/general/contributing.rst [hunk 7]
```diff
@@ -10,46 +10,11 @@ Follow these guidelines to speed up the process.
...
-
-.. code-block:: shell
-
-    git clone git@github.com:<your_fork>/django-freeradius.git
-    cd django-freeradius
-    workon radius  # activate virtualenv
-    python setup.py develop
+    Before continuing with your contributing endeavours, please read the
+    OpenWISP contributing guidelines to ensure smooth working of the
+    freeradius repository. The guidelines can be found at the following `link here <http://openwisp.io/docs/developer/contributing.html>`_
```
Comment (nemesisdesign) [inline docs/source/general/contributing.rst]: shorten to:
```
**In order to have your contribution accepted faster**, please read the
`OpenWISP contributing guidelines <http://openwisp.io/docs/developer/contributing.html>`_
and make sure to follow its guidelines.
```

File: docs/source/general/setup.rst [hunk 8]
```diff
@@ -128,8 +128,64 @@ In case you are using `openwisp-radius <https://github.com/openwisp/openwisp-rad
     ./manage.py migrate --fake openwisp_radius 0001_initial_freeradius
     ./manage.py migrate
 
-Installing for development
---------------------------
+
+Automating management commands
```
Comment (nemesisdesign) [inline docs/source/general/setup.rst]: this section should not be moved, it was good to keep it below, undo this change please

**Comment by atb00ker** [comment]
@aviral2004 are you going to complete it or should close it? :smile:

**Comment by atb00ker** [comment]
Closing as this is solved! :smile:
